src/vertical-nav.scss

```diff
@@ -16,27 +16,15 @@ $block: #{$fd-namespace}-vertical-nav;
   box-shadow: var(--sapContent_Shadow0);
   height: 100%;
 
-  &__group-header {
-    @include fd-reset();
-    @include fd-ellipsis();
-    @include fd-set-paddings-x(1rem);
-
-    height: 2.75rem;
-    background: var(--sapList_GroupHeaderBackground);
-    border-bottom: var(--sapList_BorderWidth) solid var(--sapList_GroupHeaderBorderColor);
-    color: var(--sapList_GroupHeaderTextColor);
-    display: flex;
-    align-items: flex-end;
-    font-size: var(--sapFontSize);
-    font-weight: bold;
-    line-height: 2rem;
-  }
-
   &__main-navigation {
     @include fd-reset();
   }
 
   &--condensed {
     width: 3.25rem;
   }
+
+  .#{$block}__group-header {
+    background: var(--sapList_AlternatingBackground);
+  }
```

Comment:
removed the other code as I used for base `.fd-list__group-header` and just added this class to change the background.